readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -77,6 +77,22 @@ python-install:
   # Default: []
   extra_requirements: list(str(), required=False)
 
+python-install-pipfile:
+  # The path to the project to be installed
+  pipfile: path()
+
+  # Add the --dev option to pipenv install
+  # Default: false
+  dev: bool(required=False)
+
+  # Add the --ignore-pipfile option to pipenv install
+  # Default: true
+  ignore_pipfile: bool(required=False)
+
+  # Add the --skip-lock option to pipenv install
+  # Default: false
+  skip_lock: bool(required=False)
```

Comment:
Why skip lock? See https://github.com/rtfd/readthedocs.org/issues/3181#issuecomment-430762560

Reply:
Yeah, but this will be on the implementation, we can't express that with yamale

Reply:
And, pipenv works find with both options, I think that one project only worked with the two options on, I'll check that again

Reply:
See conversation here: https://github.com/rtfd/readthedocs.org/issues/3181#issuecomment-339217578

What I take away from this is that lock file usage should not be our default as 1) it might not be checked in anyways, and 2) could be tailored to the deployment environment, not RTD.

Reply:
Ok, I can confirm, pipenv doesn't throw any error, when using both options, and the project that I mention only needs the `--skip-lock` option.

Reply:
Pipfile has python specific things too https://pipenv.readthedocs.io/en/latest/advanced/#specifying-basically-anything as reading https://github.com/pypa/pipenv/issues/857 with have the same version of python should be fine (most of the cases anyway).

So, I think we want to add `--skip-lock` too, if not, pipenv will generate a lock file when installing from Pipfile and that's slow

Reply:
So don't we want `skip_lock=true` by default?

Reply:
Nevermind, conversation was out of date! Changes match my understanding of `Pipfile` and `Pipfile.lock`